benchmarks/one_comp_mm_elim_abs/one_comp_mm_elim_abs.stan

```diff
@@ -49,11 +42,9 @@ parameters {
 }
 
 transformed parameters {
-  real C[N_t, 1];
-  {
-    real theta[3] = {k_a, K_m, V_m};
-    C = integrate_ode_bdf(one_comp_mm_elim_abs, C0, t0, times, theta, x_r, x_i);
-  }
+  vector[1] C[N_t] = ode_bdf_tol(one_comp_mm_elim_abs, C0, t0, times,
+				 1e-8, 1e-8, 1000,
+				 k_a, K_m, V_m, D, V);
```

Comment:
Can you please vectorise the likelihood? It wasn't vectorised before, but it should have been. Also: can we please swap `C_hat` to be the mean while `C` is the observed thing?  So maybe write

`C ~ lognormal(log(C_hat[,1]), sigma); `

and swap the declarations for data / transformed data. Calling C_hat the observed (with noise) is really odd to me. Thanks.